tools/scripts/scvi/scvi-train.py

```diff
@@ -0,0 +1,61 @@
+import cellxgene_census
+import tiledbsoma as soma
+import yaml
+from cellxgene_census.experimental.pp import highly_variable_genes
+
+import scvi
+
+file = "scvi-config.yaml"
+
+with open(file) as f:
+    config = yaml.safe_load(f)
+
+census = cellxgene_census.open_soma()
+
+experiment_name = "mus_musculus"
+obs_value_filter = 'tissue_general == "heart" and is_primary_data == True'
```

Comment:
We'll have to also make this more of a "test" mode

Reply:
I was thinking to add this as an optional parameter.